aws/addAppSettings.sh

```diff
@@ -33,11 +34,11 @@ while getopts ":hs:u" opt; do
             UPDATE_APPSETTINGS="true"
             ;;
         \?)
-            echo -e "\nInvalid option: -${OPTARG}"
+            cat <<< "Invalid option: -${OPTARG}" >&2
             usage
             ;;
         :)
-            echo -e "\nOption -${OPTARG} requires an argument"
+            cat <<< "Option -${OPTARG} requires an argument" >&2
```

Comment:
If you look at the issue, I referred to echo where the output is directed to a pipe. If its just an echo, then there isn't much benefit in the heredoc. The reason to use it is to avoid a subshell generated by the pipe as opposed to simple redirection.